_migrations/index.md

```diff
@@ -9,51 +9,61 @@ has_toc: false
 
 ### Traffic Replayer
+
 Acting as a traffic simulation tool, the Traffic Replayer replays recorded request traffic to a target cluster, mirroring source traffic patterns. It links original requests and their responses to those directed at the target cluster, facilitating comparative analysis.
 
-### Metadata Migration Tool
-A tool integrated into the Migration CLI that can also be used independently to migrate cluster metadata, including index mappings, index configuration settings, templates, component templates, and aliases.
+### Metadata migration tool
 
-### Reindex-from-Snapshot
-Reindexing data from an existing snapshot on Elastic Container Service (ECS) workers that coordinate the migration of documents from an existing snapshot, reindexing the documents in parallel to a target cluster.
+The Metadata migration tool integrated into the Migration CLI can be used independently to migrate cluster metadata, including index mappings, index configuration settings, templates, component templates, and aliases.
+
+### reindex-from-snapshot
+
+`reindex-from-snapshot` reindexes data from an existing snapshot on Elastic Container Service (ECS) workers that coordinate the migration of documents from an existing snapshot, reindexing the documents in parallel to a target cluster.
+
+### Target cluster
 
-### Target Cluster
 The destination cluster for migration or comparison in an A/B test.
 
-### Architecture Overview
-This architecture is based on the use of AWS cloud infrastructure, but most tools are designed to be cloud-independent. A local containerized version of this solution is also available.
+### Architecture overview
+
+The Migration assistant architecture is based on the use of an AWS cloud infrastructure, but most tools are designed to be cloud-independent. A local containerized version of this solution is also available.
 
 The design deployed in AWS is as follows: 
 
 ![Migration architecture overview]({{site.url}}{{site.baseurl}}/images/migrations/migration-architecture-overview.svg)
 
 1. Client traffic is directed to the existing cluster.
-2. An ALB with Capture Proxies relaying traffic to source while replicating to Amazon MSK.
-3. With continuous traffic capture in place, a Reindex-from-Snapshot (RFS) is initiated by the user via Migration Console.
-4. Once Reindex-from-Snapshot is complete, traffic captured is replayed from MSK by Traffic Replayer.
-5. Performance and behavior of traffic sent to source and target clusters are compared by reviewing logs and metrics.
-6. After confirming the target cluster’s functionality meets expectations the use redirects clients to new target.
+2. An Application Load Balancer (ALB) with capture proxies relays traffic to a source while replicating data to Amazon Managed Streaming for Apace Kafka (AWS MSK).
+3. With continuous traffic capture in place, a `reindex-from-snapshot` is initiated by the user via Migration Console.
```

Comment:
we should actually have a bullet point before this:

3. The user can initiate metadata migration to establish indices, templates, component templates, and aliases on the target cluster, using the Migration Console.
4. With continuous traffic capture in place, a `reindex-from-snapshot` process is initiated by the user via Migration Console. (or similar wording)